README.md

```diff
@@ -148,6 +149,11 @@ Google is providing this collection of pre-implemented Dataproc templates as a r
        ```
        bin/start.sh -- --template BIGQUERYTOGCS
        ```
+   1. #### Executing PubSub to GCS template.
```

Comment:
I would prefer to remove this part of readme as it does not adds much value.
Separate readme file is already linked on the top.